lib/logatron/basic_logger.rb

```diff
@@ -22,6 +22,10 @@ def debug(msg)
       write(format_log(msg: msg, severity: DEBUG), SEVERITY_MAP[DEBUG])
     end
 
+    def invalid_user(msg)
+      write(format_log(msg: msg, severity: INVALID_USE), SEVERITY_MAP[INVALID_USE])
```

Comment:
Or the reverse? Maybe this method is supposed to be called 

```
def invalid_use(msg)
```

which would make more sense. Either way, there is a typo.